### PR TITLE
Add to product new attribute "color".

### PR DESCRIPTION
**Pull request by Wojdylak**

### CODE REVIEW

File: config/doctrine/Product/Product.orm.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping">
+    <entity name="App\Entity\Product\Product" table="sylius_product">
+        <field name="color" type="string"/>
+    </entity>
+</doctrine-mapping>
```
Comment (Roshyo) [inline config/doctrine/Product/Product.orm.xml:7]: All the mappings in current Sylius Standard are done in annotation, not in XML

File: config/packages/_sylius.yaml [hunk 2]
```diff
@@ -28,18 +28,6 @@ sylius_addressing:
...
-        product:
-            attribute:
-                classes:
-                    model: App\Entity\Product\ProductAttribute
-                translation:
-                    classes:
-                        model: App\Entity\Product\ProductAttributeTranslation
-            attribute_value:
-                classes:
-                    model: App\Entity\Product\ProductAttributeValue
```
Comment (Roshyo) [inline config/packages/_sylius.yaml:42]: Why are you removing all the overrides of resources ? What's the point ?

File: config/services.yaml [hunk 3]
```diff
@@ -33,3 +33,8 @@ services:
     App\Controller\:
         resource: '../src/Controller'
         tags: ['controller.service_arguments']
+
+    App\Form\Type\ProductType:
+        tags: [ 'form.type' ]
+        arguments:
+            - 'App\Entity\Product\Product'
```
Comment (Roshyo) [inline config/services.yaml:40]: You should rather use a FormExtension as the ProductForm already exists in Sylius.

File: src/Entity/Product/Product.php [hunk 4]
```diff
@@ -4,18 +4,31 @@
...
- * @ORM\Entity
- * @ORM\Table(name="sylius_product")
- */
 class Product extends BaseProduct
 {
+    const COLOR_RED = 'RED';
+    const COLOR_BLUE = 'BLUE';
+    const COLOR_GREEN = 'GREEN';
+
+    /** @var string $color */
```
Comment (Roshyo) [inline src/Entity/Product/Product.php:17]: This docblock is not needed

File: src/Form/Type/ProductType.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+final class ProductType extends AbstractResourceType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add(
+                'color',
+                ChoiceType::class,
+                [
+                    'label' => 'color.label',
```
Comment (Roshyo) [inline src/Form/Type/ProductType.php:21]: shouldn't this be under some `sylius.ui.` ?

**Review by Roshyo — COMMENTED** [review]
I think I'm missing the point of this PR, sorry.

**Comment by Wojdylak** [comment]
It was a mistake.

**Pull request closed without merging**